yoti_python_sdk/sandbox/token.py

```diff
@@ -0,0 +1,128 @@
+from yoti_python_sdk.sandbox.attribute import SandboxAttribute
+from yoti_python_sdk import config
+import base64
+
+
+class YotiTokenResponse(object):
+    def __init__(self, token):
+        self.__token = token
+
+    @property
+    def token(self):
+        return self.__token
+
+
+class YotiTokenRequest(object):
+    def __init__(self, remember_me_id=None, sandbox_attributes=None):
+
+        if remember_me_id is None:
+            remember_me_id = ""
+
+        if sandbox_attributes is None:
+            sandbox_attributes = []
+
+        self.remember_me_id = remember_me_id
+        self.sandbox_attributes = sandbox_attributes
+
+    def __dict__(self):
+        return {
+            "remember_me_id": self.remember_me_id,
+            "profile_attributes": self.sandbox_attributes,
+        }
+
+    @staticmethod
+    def builder():
```

Comment:
As a matter of code quality, do we want to require doc comments on public methods?

Reply:
This is a dependency from #100 so I'll go through all of the Sandbox stuff in that PR and add docstrings, and then once that PR has been merged I'll rebase this branch on development

Reply:
Fixed in #100